components/Chat.tsx

```diff
@@ -0,0 +1,55 @@
+import { View, Text, StyleSheet, Image } from 'react-native';
+import React from 'react';
+
+type ChatProps = {
+  Name: string;
+  Image: string;
+  Message: string;
+};
+
+export default function Chat(props: ChatProps) {
+  let pic = {
```

Comment:
Why use let when it's not reassigned? Just use `const` instead for more clarity
```suggestion
  const pic = {
```